com.unity.cinemachine/Editor/Utility/CmCameraInspectorUtility.cs

```diff
@@ -90,8 +90,10 @@ void UpdateCameraStatus()
             if (m_NavelGazeMessage != null)
             {
                 CameraState state = Target.State;
-                bool isNavelGazing = Target.PreviousStateIsValid 
-                    && state.HasLookAt() && (state.ReferenceLookAt - state.GetCorrectedPosition()).AlmostZero();
+                bool isNavelGazing = Target.PreviousStateIsValid && state.HasLookAt() &&
+                    (state.ReferenceLookAt - state.GetCorrectedPosition()).AlmostZero() &&
+                    (Target.GetCinemachineComponent(CinemachineCore.Stage.Body) != null ||
```

Comment:
this condition is too strong.  Should only check for Aim not null.  Body doesn't matter.